main.tf

```diff
@@ -0,0 +1,65 @@
+
+data "template_file" "nomad_job_redash_server" {
+  template = file("${path.module}/conf/nomad/redash_server.hcl")
+}
+
+data "template_file" "nomad_job_redash_worker" {
+  template = file("${path.module}/conf/nomad/redash_worker.hcl")
+}
+
+data "template_file" "nomad_job_redash_scheduler" {
+  template = file("${path.module}/conf/nomad/redash_scheduler.hcl")
+}
+
+data "template_file" "nomad_job_postgres" {
+  template = file("${path.module}/conf/nomad/postgres.hcl")
+}
+
+data "template_file" "nomad_job_redis" {
+  template = file("${path.module}/conf/nomad/redis.hcl")
+}
+
+data "template_file" "nomad_job_email" {
+  template = file("${path.module}/conf/nomad/email.hcl")
+}
+
+resource "nomad_job" "nomad_job_redash_server" {
+  jobspec = data.template_file.nomad_job_redash_server.rendered
+  detach = false
+  depends_on = [
+    "nomad_job.nomad_job_redis",
+    "nomad_job.nomad_job_postgres",
+    "nomad_job.nomad_job_email",
+  ]
+}
+
+resource "nomad_job" "nomad_job_redash_worker" {
+  jobspec = data.template_file.nomad_job_redash_worker.rendered
+  detach = false
+  depends_on = [
+    "nomad_job.nomad_job_redash_server",
+  ]
+}
+
+resource "nomad_job" "nomad_job_redash_scheduler" {
+  jobspec = data.template_file.nomad_job_redash_scheduler.rendered
+  detach = false
+  depends_on = [
+    "nomad_job.nomad_job_redash_server",
+  ]
+}
+
+resource "nomad_job" "nomad_job_postgres" {
+  jobspec = data.template_file.nomad_job_postgres.rendered
+  detach = false
+}
```

Comment:
Why not use: https://github.com/fredrikhgrelland/terraform-nomad-postgres/releases/tag/0.0.1

Reply:
This was just a POC to see if it all worked, that's why. We are now adding healthchecks to everything in a new branch. 
Would you like us to use his module in this first PR?

Reply:
nope. Just pointing it out.